src/SprykerMiddleware/Zed/Process/Business/Log/Config/MiddlewareLoggerConfig.php

```diff
@@ -67,7 +67,7 @@ protected function createStdErrStreamHandler(): StreamHandler
             'php://stderr',
             $this->loggerSettings->getVerboseLevel()
         );
-        $formatter = new LogstashFormatter(static::NAME);
+        $formatter = new LogstashFormatter(static::CHANNEL_NAME);
```

Comment:
should be $this->getChannelName() called instead?

Reply:
I saw it in your PR, so it will be merged with your changes